guardian/scribe-plugin-noting
# Flag and Correct

This PR adds the ability to create flags and corrects.

![flag-and-correct](https://cloud.githubusercontent.com/assets/1975139/6083072/0073576e-ae1e-11e4-8695-f8574604a707.gif)

### QA 
- Create a note then try to create a flag within the note (F9). Ensure nothing is created.
- Create a flag. Ensure a flag is created.
- Click the created flag. Ensure it converts to a correct.
- Try to create a note (F10). Ensure a note is not created.
